Makefile

```diff
@@ -366,4 +366,8 @@ pod-failover-test-containers:
 
 .PHONY: upgrade-test
 upgrade-test:
-	go test -v -timeout=0 ${GOTAGS} ./test/upgrade
+	go test -v -timeout=0 ${GOTAGS} ./test/upgrade $(GINKGO_FLAGS)
```

Comment:
`GINKGO_FLAGS` is mainly used to filter the multi/single az tests in e2e. We don't need those focus filters for these tests., Why do we pass the flags here? If you want to use the -v flag for goinkgo, I think we should use it directly instead of using `GINKGO_FLAGS` value.

Reply:
This is for potential future changes if we decide to test unplanned failover with multi zone scenarios. I added the flags to upgrade test as well just for the sake of it :p.

Reply:
I just noticed that we run this test with both single/multi az!